doc/FormattingDecisionComments/columns.html

```diff
@@ -0,0 +1,86 @@
+
+<!-- generated by Makefile (python3 columns.py) -->
+
+<html>
+<head>
+  <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+</head>
+
+<body>
+
+
+<div id="myPlot"></div>
+</div>
+<script>
+var data = [
+
+{
+x: [17, 19, 20, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 51, 56, 76],
+y: [1.580135440180587, 1.3544018058690745, 3.6681715575620766, 2.0316027088036117, 1.580135440180587, 1.18510158013544, 1.7494356659142212, 1.693002257336343, 1.9751693002257336, 2.4266365688487586, 7.618510158013544, 2.6523702031602707, 1.580135440180587, 2.539503386004515, 2.4266365688487586, 1.1286681715575622, 1.4108352144469527, 5.756207674943567, 5.586907449209932, 2.200902934537246, 2.2573363431151243, 1.4108352144469527, 1.580135440180587, 2.144469525959368, 2.144469525959368, 1.4672686230248306, 2.37020316027088, 17.099322799097067, 1.4108352144469527, 1.2415349887133182, 2.0316027088036117, 1.0158013544018059],
+mode: 'markers',
+type: 'scatter',
+name: 'OTP'
+},
+{
+x: [68, 72],
+y: [28.0, 20.0],
+mode: 'markers',
+type: 'scatter',
+name: 'kazoo'
+},
+{
+x: [],
+y: [],
+mode: 'markers',
+type: 'scatter',
+name: 'Inaka'
+},
+{
+x: [],
+y: [],
+mode: 'markers',
+type: 'scatter',
+name: 'MongooseIM'
+},
+{
+x: [],
+y: [],
+mode: 'markers',
+type: 'scatter',
+name: 'ejabberd'
+},
+{
+x: [9, 11, 17, 18, 20, 21, 22, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 51, 52, 53, 56, 58, 59, 60, 61, 68, 92],
+y: [2.593192868719611, 1.7017828200972447, 1.4181523500810373, 1.8638573743922204, 1.9448946515397083, 1.2155591572123177, 1.4181523500810373, 1.6207455429497568, 1.4586709886547813, 2.1880064829821717, 2.1880064829821717, 2.674230145867099, 2.3095623987034037, 1.2155591572123177, 1.0940032414910859, 4.416531604538087, 1.8233387358184765, 1.5802269043760129, 1.8233387358184765, 2.674230145867099, 2.512155591572123, 2.025931928687196, 1.8233387358184765, 3.9708265802269045, 1.7828200972447326, 3.282009724473258, 1.2965964343598055, 1.7017828200972447, 1.6207455429497568, 1.2560777957860616, 1.6612641815235007, 1.9448946515397083, 1.1345218800648298, 1.539708265802269, 1.3371150729335495, 1.2965964343598055, 1.1750405186385737, 1.1345218800648298, 2.552674230145867, 2.025931928687196, 1.6612641815235007, 1.2560777957860616, 1.5802269043760129],
+mode: 'markers',
+type: 'scatter',
+name: 'WhatsApp'
+},
+
+];
+
+var layout =
+
+{
+
+title: 'Percentage of comments at column number',
+
+
+xaxis: {
+    title: 'comment starts at column number'
+},
+yaxis: {
+    title: 'percentage of comments'
+}
+};
+
+
+
+Plotly.newPlot('myPlot', data, layout);
+
+</script>
+
+
+</body>
+</html>
```

Comment:
the html plots are a nice touch, but it's rather hard to see them from github. Should we worry about that?

Reply:
I tried gnuplot and matplotlib but gave up.
Both had issues installing or crashed while running, so I decided to limit the time that I spent on this.
It is inconvenient to view on github though :(
You have to download the raw file to view, I am sorry.
I hope it is okay this way though.